AnkiHubSoftware/ankihub_addon
# Add multi threads support to images download

## Proposed changes
This PR adds multi threads support to the download images workflow. In the background task now we will create threads to handle the image download on s3 (managed by ThreadPoolExecutor class).


## How to reproduce

1. Subscribe to a deck on AnkiHub
2. Add the subscribed deck in the "Subscribed Decks" menu  option in the addon
3. Click on "Sync with AnkiHub" menu  option in the addon
4. Check if the notes are with the images
